From 7702b72a0de8664645c7524c828fd452abf998aa Mon Sep 17 00:00:00 2001
From: Philipp Kern <pkern@debian.org>
Date: Fri, 27 Mar 2015 11:27:08 +0100
Subject: [PATCH] Do not interpret a buffer as a format string.

---
 src/graphics/C4Shader.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/graphics/C4Shader.cpp b/src/graphics/C4Shader.cpp
index 2acd227..5216a0a 100755
--- a/src/graphics/C4Shader.cpp
+++ b/src/graphics/C4Shader.cpp
@@ -505,7 +505,7 @@ void C4Shader::DumpInfoLog(const char *szWhat, GLhandleARB hShader)
 	// Terminate, log
 	pBuf[iActualLength] = '\0';
 	ShaderLogF("  gl: Compiling %s:", szWhat);
-	ShaderLogF(pBuf);
+	ShaderLogF("%s", pBuf);
 	delete[] pBuf;
 }
 
-- 
2.1.4

